src/Nethermind/Nethermind.Core/IBatch.cs

```diff
@@ -5,5 +5,6 @@
 
 namespace Nethermind.Core
 {
-    public interface IBatch : IDisposable, IKeyValueStore { }
+    public interface IBatch : IDisposable, IKeccakValueStore { }
```

Comment:
Does not seems like a good idea. Should've made a different interface `IKeccakBatch` instead.